ocaml/database/block_device_io.ml

```diff
@@ -325,16 +325,20 @@ let listen_on sock =
   s
 
 let accept_conn s latest_response_time =
-  let now = Unix.gettimeofday () in
```

Comment:
Commit contains:
> TODO: needs some extra testing of block_device_io, which doesn't many suitable tests.

Has this been done in some way?

Reply:
There is 1 XenRT test capable of detecting breakages in the redolog, the one that got added for HA vTPM testing.
(all other HA XenRT tests used to pass with a completely broken redolog in the past).

There is a test for epoll on block devices part of the stdext quickcheck tests, although it requires root so it is not run in the CI.
I'll add some code to the XAPI quicktests so we can run some of these fuzzers in Dom0.

I think it is still worthwhile trying to add a separate test for the redolog itself that runs as part of the quicktests (it needs a blockdevice, and setting one up even in loopback mode requires root, so can't be done in the CI).